### PR TITLE
PAAS-1875 handle not expected cases

### PR DESCRIPTION
**Pull request by laurentfufu**
JIRA issue: https://jira.jahia.org/browse/PAAS-1875

Short description:

### CODE REVIEW

File: packages/one-shot/paas-1875-upgrade-distributed-sessions.yaml [hunk 1]
```diff
@@ -110,11 +134,32 @@ actions:
...
-    - setTargetModuleVersions
+    - setTargetModuleVersion
     - getDistSessStatus: proc
-    - if((response.active) && ("${response.version}" != "${globals.distSessVersion}")):
-        - enableKarafLogin: proc
+    - setGlobals:
+        DS_count: ${response.resp.length}
+        stringList: "${response.resp}"
+    - if(${response.resp.length} == 0):
+        - raiseError: "No Distribution Sessions module here"
```
Comment (kevinvandel) [inline packages/one-shot/paas-1875-upgrade-distributed-sessions.yaml:148]: *Distributed

File: packages/one-shot/paas-1875-upgrade-distributed-sessions.yaml [hunk 2]
```diff
@@ -12,50 +12,74 @@ globals:
...
-              if(active=="true"){ \
-                printf "{\"version\": \"%s\", \"active\": %s, \"status\": %s }",version,active,status \
-              } \
-              else{ \
-              printf "{\"active\": false, \"status\": %s}",status \
-              } \
-            }' \
-          /data/digital-factory-data/bundles-deployed/*/bundle.info
-
+        #   - array like object with on element by module version found
```
Comment (kevinvandel) [inline packages/one-shot/paas-1875-upgrade-distributed-sessions.yaml:35]: *_one_ element

File: packages/one-shot/paas-1875-upgrade-distributed-sessions.yaml [hunk 3]
```diff
@@ -139,6 +184,10 @@ actions:
             ${globals.karafConsole} cluster:bundle-start default distributed-sessions/${globals.distSessVersion}
             ${globals.karafConsole} cluster:bundle-uninstall default distributed-sessions/$old_version
         - disableKarafLogin: proc
+        - checkStat
+    - else:
+        - disableKarafLogin: proc
+        - log: "Looks like that all is already good here, full status: ${globals.stringList}"
```
Comment (kevinvandel) [inline packages/one-shot/paas-1875-upgrade-distributed-sessions.yaml:190]: -that

File: packages/one-shot/paas-1875-upgrade-distributed-sessions.yaml (hunk 1 above)
Comment (laurentfufu) [reply packages/one-shot/paas-1875-upgrade-distributed-sessions.yaml]: fixed

File: packages/one-shot/paas-1875-upgrade-distributed-sessions.yaml (hunk 2 above)
Comment (laurentfufu) [reply packages/one-shot/paas-1875-upgrade-distributed-sessions.yaml]: fixed

File: packages/one-shot/paas-1875-upgrade-distributed-sessions.yaml (hunk 3 above)
Comment (laurentfufu) [reply packages/one-shot/paas-1875-upgrade-distributed-sessions.yaml]: fixed

**Review by Ronan-Gorain — APPROVED** [review]

**Review by kevinvandel — APPROVED** [review]

**Pull request merged**